app/javascript/store/modules/admin/actions.js

```diff
@@ -6,8 +6,11 @@ import {
 
 export default {
     [UPDATE_DEFAULT_TEAM](
-      {}, { teamId, organization }) {
+      {}, { teamId, organization, froggoTeam }) {
       axios
-        .put(`/api/organizations/${organization.id}/update`, { 'default_team_id': teamId });
+        .put(`/api/organizations/${organization.id}/update`, {
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected indentation of 6 spaces but found 8   indent